tink/internal/client_test/hardware_test.go

```diff
@@ -0,0 +1,123 @@
+/*
```

Comment:
I wonder, why `tink/internal` for directory structure and not `internal/tink`?


Also, maybe `tinkerbell` instead of `tink`?